frontend/src/store/modules/activity.ts

```diff
@@ -83,23 +84,39 @@ export const useActivityStore = defineStore("activity", {
       this.setActivityListForUser({ userId, activityList });
       return activityList;
     },
-    async fetchActivityListForIssue(issueId: IssueId) {
-      const queryList = [
-        "typePrefix=bb.issue.",
-        `container=${issueId}`,
-        `order=ASC`,
-      ];
-      const data = (await axios.get(`/api/activity?${queryList.join("&")}`))
-        .data;
-      const activityList: Activity[] = data.data.map(
+    async fetchActivityList(params: {
+      typePrefix: string;
+      container: number | string;
+      order: "ASC" | "DESC";
+      limit?: number;
+    }) {
+      const url = `/api/activity?${stringify(params)}`;
+      const response = (await axios.get(url)).data;
+      const activityList: Activity[] = response.data.map(
         (activity: ResourceObject) => {
-          return convert(activity, data.included);
+          return convert(activity, response.included);
         }
       );
-
-      this.setActivityListForIssue({ issueId, activityList });
       return activityList;
     },
+    async fetchActivityListForIssue(issueId: IssueId) {
+      const listForIssue = await this.fetchActivityList({
+        typePrefix: "bb.issue.",
+        container: issueId,
+        order: "ASC",
+      });
+      const listForPipeline = await this.fetchActivityList({
+        typePrefix: "bb.pipeline.",
+        container: issueId,
+        order: "ASC",
+      });
```

Comment:
Can we use `Promise.all` here to fetch these two lists concurrently?

Reply:
Good idea